portal-kernel/src/com/liferay/portal/kernel/comment/BaseDiscussionPermission.java

```diff
@@ -35,7 +35,8 @@ public void checkAddPermission(
 				permissionChecker, companyId, groupId, className, classPK)) {
 
 			throw new PrincipalException.MustHavePermission(
-				0, className, classPK, ActionKeys.ADD_DISCUSSION);
+				permissionChecker, className, classPK,
```

Comment:
@W2jx This commit says "use correct user id" but it changes "0" to `permissionChecker`. Could you update the message to be more clear?